vm/internal/handler/clone.go

```diff
@@ -29,32 +30,38 @@ func (h *Handler) CloneVolume(ctx echo.Context) error {
 	log.Infof("volumeName: %s", volumeName)
 	log.Infof("destVolume: %s", destVolume)
 
+	cli, err := h.DockerClient()
+	if err != nil {
+		log.Error(err)
+		return ctx.String(http.StatusInternalServerError, err.Error())
+	}
+
 	defer func() {
 		h.ProgressCache.Lock()
 		delete(h.ProgressCache.m, volumeName)
 		h.ProgressCache.Unlock()
-		_ = backend.TriggerUIRefresh(ctx.Request().Context(), h.DockerClient)
+		_ = backend.TriggerUIRefresh(ctx.Request().Context(), cli)
 	}()
 
 	h.ProgressCache.Lock()
 	h.ProgressCache.m[volumeName] = "clone"
 	h.ProgressCache.Unlock()
 
-	err := backend.TriggerUIRefresh(ctx.Request().Context(), h.DockerClient)
+	err = backend.TriggerUIRefresh(ctx.Request().Context(), cli)
 	if err != nil {
```

Comment:
nit: Isn't it better to avoid a reassignment or `err` with the following?
```suggestion
	if err := backend.TriggerUIRefresh(ctx.Request().Context(), cli); err != nil {
```